packages/apollo-engine-reporting/src/agent.ts

```diff
@@ -194,6 +201,8 @@ export interface AddTraceArgs {
   schemaHash: string;
   queryString?: string;
   documentAST?: DocumentNode;
+  gqlValidationFailure: boolean;
```

Comment:
do we really use "gql" much?

Reply:
^^

Reply:
Will change to `graphqlValidationFailure`